packages/application-config/src/process-config.ts

```diff
@@ -129,6 +129,7 @@ const processConfig = ({
                 : appConfig.env.development.initialProjectKey,
             teamId: appConfig.env.development?.teamId,
             oAuthScopes: appConfig.oAuthScopes,
+            additionalOAuthScopes: appConfig?.additionalOAuthScopes,
```

Comment:
Let me pin my comment here.

This file calls `transformCustomApplicationConfigToData` function (defined [here](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/application-config/src/transformers.ts)) which returns an object containing [`permissions` array](https://github.com/commercetools/merchant-center-application-kit/blob/91c0da4f9afa92e44a22c8c3af38f545488d5e44/packages/application-config/src/transformers.ts#L38:L47).

It seems this property is not used anywhere in the codebase, and I'm wondering if that should be somehow updated (or perhaps even removed)? I guess I'm missing some context 🤔